code/mod_table/mod_table.cpp

```diff
@@ -396,20 +397,68 @@ void parse_mod_table(const char *filename)
 			stuff_boolean(&Render_player_mflash);
 		}
 
+		if (optional_string("$Glowpoint nebula visibility factor:")) {
+			stuff_float(&Neb2_fog_visibility_glowpoint);
+		}
+
+		if (optional_string("$Shield nebula visibility factor:")) {
+			stuff_float(&Neb2_fog_visibility_shield);
+		}
+
+		if (optional_string("$Thruster nebula visibility factor:")) {
+			stuff_float(&Neb2_fog_visibility_thruster);
+		}
+
 		if (optional_string("$Beams affected by nebula visibility:")) {
 			stuff_boolean(&Neb_affects_beams);
+
+			if (optional_string("+Constant visibility factor:")) {
+				stuff_float(&Neb2_fog_visibility_beam_const);
+			}
+
+			if (optional_string("+Size visibility factor:")) {
+				stuff_float(&Neb2_fog_visibility_beam_size_factor);
```

Comment:
This is acceptable, but I would prefer something like `+Visibility factor slope` and `+Visibility factor intercept` (or `offset`).  The term "constant" doesn't sit right for me here, and the slope-intercept terms have the advantage of being standard math terms that match the common `y = mx + b` formula.

Reply:
Maybe, but I would also like people who aren't very good at understanding things mathematically to understand this as well, and I worry "slope" and "intercept" would be too confusing in this context, hopefully as opposed to "constant" (the amount you always get) and "size" (i.e. the amount that size affects it). From the descriptions I'll give it should hopefully be clear either way what its mathematical behavior should be.

Reply:
How about `+Scaled visibility factor`?